texture.js

```diff
@@ -228,6 +228,77 @@ function renderAllTextures(element, game) {
 	});
 }
 
+// ...
+// XXX hardcoded heretic
+function drawFlat(f) {
+    fn = "heretic/flats/"+f.toLowerCase()+".png";
+    var img = document.createElement("img");
+    img.src = fn;
+    return img;
+}
+
+// ...
+function parseFlats(wadinfo) {
+    var lines = wadinfo.split("\n");
+    var i;
+    var flats = []; f = null;
+    // skip until [flats]
```

Comment:
I'd suggest just a simple text file containing the list of flats, rather than bother trying to parse wadinfo.txt.

Reply:
Certainly easier from the JS side of things but then I/we'd have to generate said text file ourselves... I suppose that's a one-time thing most likely.

Reply:
Exactly. `ls *.png > flags.txt` doesn't seem like a huge amount of effort.

Reply:
Oh yeah of course!